docs/providers/tencent/cli-reference/install.md

```diff
@@ -0,0 +1,52 @@
+<!--
+title: Serverless Framework Commands - Tencent-SCF - Install
+menuText: install
+menuOrder: 2
+description: Install pre-written Tencent-SCF Functions, Events and Resources with the Serverless Framework
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/providers/tencent/cli-reference/install/)
+
+<!-- DOCS-SITE-LINK:END -->
+
+# Tencent-SCF - Install
+
+Installs a service from a GitHub URL in the current working directory.
+
+```bash
+serverless install --url https://github.com/some/service
+```
+
+## Options
+
+- `--url` or `-u` The services Git URL. **Required**.
+- `--name` or `-n` Name for the service.
+
+## Examples
+
+### Installing a service from a GitHub URL
+
+```bash
+serverless install --url https://github.com/tencentcloud/serverless/tencent-nodejs
```

Comment:
Hi @tinafangkunding , this link seems to be broken, could you please help fix this?

Reply:
> Hi @tinafangkunding , this link seems to be broken, could you please help fix this?

Yes sure, I'll fix that ASAP

Reply:
Already fix that, thanks!